globus_sdk/transfer/client.py

```diff
@@ -334,6 +334,67 @@ def submit_delete(self, data):
         """
         return self.post('/delete', data)
 
+    def make_submit_transfer_data(self, source_endpoint, dest_endpoint,
+                                  transfer_items, label, sync_level=None,
+                                  **kwargs):
```

Comment:
I don't know. If it's required in the SDK I can see people saying, "I don't want a label, `label` is a string, therefore I'm going to pass in `''`."

In our web app we actually check for an empty string and convert it to `null` before sending the request off to the API (because if you're taking input from a user it's really easy to end up with a "no input" -> empty string -> transfer api error situation).

So unless there's a compelling reason to make `label` required it seems like it might cause hassle for no reason.